src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -1082,7 +1201,7 @@ public static void readFromNBT(@NotNull final NBTTagCompound compound, @NotNull
                     (int) ((Math.cos(45.0 / HALF_A_CIRCLE * Math.PI) * Configurations.gameplay.workingRangeTownHall) / BLOCKS_PER_CHUNK);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 45.0 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)